src/components/molecules/PoolTransactions.tsx

```diff
@@ -61,28 +79,35 @@ const txHistoryQuery = gql`
     }
   }
 `
-async function getSymbol(ocean: Ocean, tokenAddress: string) {
-  const symbol =
-    ocean.pool.oceanAddress.toLowerCase() === tokenAddress.toLowerCase()
-      ? 'OCEAN'
-      : await ocean.datatokens.getSymbol(tokenAddress)
+
+interface Datatoken {
+  symbol: string
+}
+
+function getSymbol(tokenId: Datatoken) {
+  const symbol = tokenId === null ? 'OCEAN' : tokenId.symbol
 
   return symbol
 }
 
 async function getTitle(
-  ocean: Ocean,
   row: TransactionHistoryPoolTransactions,
   locale: string
 ) {
-  let title = ''
+  const source = axios.CancelToken.source()
+
+  const did = web3.utils
+    .toChecksumAddress(row.poolAddress.datatokenAddress)
+    .replace('0x', 'did:op:')
+  const ddo = await retrieveDDO(did, source.token)
```

Comment:
We don't need to retrieve the `ddo` anymore if we get the token symbol from the subgraph

Reply:
True, I removed it from the "Setup" case so we really don't need it anymore.